index.html

```diff
@@ -226,76 +226,39 @@ <h1>The Parameter Registry</h1>
 
   <section>
     <h1>The Property Registry</h1>
-
     <p>
-This table summarizes the globally registered properties that can be provided
-as properties in a DID Document.
+      The following section defines the properties in a DID document.
     </p>
-
-    <table class="simple">
-      <thead>
-        <tr>
-          <th>Property</th>
-          <th>Description</th>
-        </tr>
-      </thead>
-
-      <tbody>
-        <tr>
-          <td>
-TBD
-          </td>
-          <td>
-            <div>DESCRIPTION_GOES_HERE</div>
-            <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-            <pre>
-"@context": [
-  "https://www.w3.org/ns/did/v1",
-  "https://CONTEXT_GOES_HERE/v1"
-]
-            </pre>
-          </td>
-        </tr>
-      </tbody>
-    </table>
-  </section>
-
-
     <section>
-      <h1>The Value Registry</h1>
-
-      <p>
-  This table summarizes the globally registered values that can be used as
-  property values in a DID Document.
-      </p>
-
-      <table class="simple">
-        <thead>
-          <tr>
-            <th>Value</th>
-            <th>Description</th>
-          </tr>
-        </thead>
-
-        <tbody>
-          <tr>
-            <td>
-  TBD
-            </td>
-            <td>
-              <div>DESCRIPTION_GOES_HERE</div>
-              <div><a href="">SPECIFICATION_GOES_HERE</a></div>
-              <pre>
-  "@context": [
-    "https://www.w3.org/ns/did/v1",
-    "https://CONTEXT_GOES_HERE/v1"
-  ]
-              </pre>
-            </td>
-          </tr>
-        </tbody>
-      </table>
+      <h2><dfn data-lt="publicKey" data-dfn-type="dfn" id="publicKey">publicKey</dfn></h2>
+      <p>An array of public keys. A public key is a verification method. Public keys are used for digital signatures, encryption and other cryptographic operations, which in turn are the basis for purposes such as authentication.</p>
+      <pre class="example">
+        {
+          "id": "did:example:123",
+          "publicKey": [
+            {
+              "id": "did:example:123#ZC2jXTO6t4R501bfCXv3RxarZyUbdP2w_psLwMuY6ec",
+              "type": "Ed25519VerificationKey2018",
+              "controller": "did:example:123",
+              "publicKeyJwk": {
```

Comment:
Please use publicKeyBase58 here as we've tried using this as a base example before and it immediately makes people jump to "Wait, why is there a `Ed25519VerificationKey2018` value and then the JWK repeats it by doing `Ed25519`.

Reply:
I don't agree with this, we have reached consensus as a group that publicKeyJwk is a valid representation for all formats.

Reply:
When having to choose between an example that might confuse people and an example that will probably not confuse people, we should chose the one that will probably not confuse people. There will be a publicKeyJwk property and you can use this example there.

Reply:
but i will need to provide a `publicKeyPem` or `publicKeyJwk` or some other for the example of what is a `publicKey`... an example of a publicKey will need to include example of key material no?

Reply:
Yes, which is why I said "Please use publicKeyBase58 here" :) ... You're using an `Ed25519VerificationKey2018` and the spec says encode that as either a JWK or a `publicKeyBase58` value. You encoded it as a JWK... I'd like us to encode it as a publicKeyBase58 value because it'll be less confusing to people reading the example.

Reply:
I don't agree, JWK is less confusing than base58... This is the latest draft I can find for base58... https://tools.ietf.org/id/draft-msporny-base58-00.txt

JWK is defined by here: https://tools.ietf.org/html/rfc7517

Which format is more mature  / suitable for an example?

Reply:
I have used base58, can we resolve this?